### PR TITLE
Added info to students, images, and blog sections

### PR DESCRIPTION
**Pull request by ghost**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- I added my personal information about myself in the student section. Also, I updated the blog section by adding my personal blog including article, link, and image. Upload my photo and the image of the blog in the image file.

live link: https://github.com/balderasdiana27/gci16.fossasia.org

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -646,4 +652,4 @@
   facebook: andrewjunggg
   bio: 16 years old. Bandung, Indonesia.
   lat: -6.904950
-  lng: 107.612179
```
Comment (abishekvashok) [inline _data/students.yml:649]: Can you state why have updated other students info? I hope it's a typo so please revert this change.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please fix as stated and give a live link or a screenshot also and if possible please squash all the commits into one @balderasdiana27

File: _data/blogs.yml [hunk 2]
```diff
@@ -35,7 +35,13 @@
   blog_img: rhemon-blog.png
   article: FOSSASIA, me and Open source
   article_link: https://rhemon.github.io/blog/gci/2016/11/30/00-fossassia-me-what-why/
-
```
Comment (robbyoconnor) [inline _data/blogs.yml:38]: Add this space back please

File: _data/students.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/students.yml:649]: Revert this change please

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
You are changing another student's entry and this is not okay.

**Comment by robbyoconnor** [comment]
@balderasdiana27 merged this and reverted some of your changes to another student's geo coordinates.
